### PR TITLE
Jaeger deprecation warning

### PR DESCRIPTION
**Pull request by rubvs**
Update Jaeger integration docs with a deprecation warning.

### CODE REVIEW

File: docs/en/observability/apm/jaeger-integration.asciidoc [hunk 1]
```diff
@@ -5,6 +5,8 @@
 <titleabbrev>Integrate with Jaeger</titleabbrev>
 ++++
 
+WARNING: Jaeger is deprecated and will be removed in a future version from Elastic APM, since OTEL has removed all Jaeger exporters from their https://github.com/open-telemetry/opentelemetry-specification/pull/2858[specification].
```
Comment (simitt) [inline docs/en/observability/apm/jaeger-integration.asciidoc:8]:
```suggestion
WARNING: Support for Jaeger is deprecated and will be removed in a future version from Elastic APM. https://www.jaegertracing.io/docs/1.35/client-libraries/[Jaeger clients are deprecated] in favour of OpenTelemetry SDKs and OpenTelemetry has removed all Jaeger exporters from their https://github.com/open-telemetry/opentelemetry-specification/pull/2858[specification].
```

**Review by simitt — COMMENTED** [review]
Left a minor suggestion.

**Review by 1pkg — APPROVED** [review]

File: docs/en/observability/apm/jaeger-integration.asciidoc [hunk 2]
```diff
@@ -5,6 +5,8 @@
 <titleabbrev>Integrate with Jaeger</titleabbrev>
 ++++
 
+WARNING: Support for Jaeger is deprecated and will be removed in a future version from Elastic APM. https://www.jaegertracing.io/docs/1.35/client-libraries/[Jaeger clients are deprecated] in favour of OpenTelemetry SDKs and OpenTelemetry has removed all Jaeger exporters from their https://github.com/open-telemetry/opentelemetry-specification/pull/2858[specification].
```
Comment (colleenmcginnis) [inline docs/en/observability/apm/jaeger-integration.asciidoc:8]:
```suggestion
WARNING: Support for Jaeger is deprecated and will be removed in a future version from Elastic APM. https://www.jaegertracing.io/docs/1.35/client-libraries/[Jaeger clients are deprecated] in favor of OpenTelemetry SDKs, and OpenTelemetry has removed all Jaeger exporters from their https://github.com/open-telemetry/opentelemetry-specification/pull/2858[specification].
```

**Review by colleenmcginnis — APPROVED** [review]
✨

**Pull request merged**